mirage/models/conversation.js

```diff
@@ -1,7 +0,0 @@
-import { Model, belongsTo, hasMany } from 'ember-cli-mirage';
-
-export default Model.extend({
-  message: belongsTo(),
-  user: belongsTo(),
-  conversationParts: hasMany()
-});
```

Comment:
Why is this being removed?

Reply:
https://github.com/samselikoff/ember-cli-mirage/pull/1056

So mirage can auto generate mirage models based off of the ember-data model, avoiding mismatches b/w the two.

Lmk what you think?  We can keep slowly removing these as needed or all of them in a separate PR.

Reply:
Ah okay thanks. Maybe open a separate issue so others are tracking that’s the case as a point of reference?